Shared/AuthenticationKeychainInfo.swift

```diff
@@ -72,7 +72,9 @@ open class AuthenticationKeychainInfo: NSObject, NSCoding {
 
     public required init?(coder aDecoder: NSCoder) {
         self.lastPasscodeValidationInterval = aDecoder.decodeAsDouble(forKey: "lastPasscodeValidationInterval")
-        self.lockOutInterval = aDecoder.decodeAsDouble(forKey: "lockOutInterval")
+        if let lockOutInterval = aDecoder.decodeObject(forKey: "lockOutInterval") as? NSNumber {
```

Comment:
Is this subject to the issues we had when we did the Swift 3.0 migration?

http://stackoverflow.com/questions/38124480/nskeyedarchiver-does-not-work-in-swift-3-xcode-8/40034694#40034694